libs/@local/hashql/core/Cargo.toml

```diff
@@ -14,7 +14,7 @@ hashql-diagnostics = { workspace = true, public = true }
 anstyle            = { workspace = true, public = true }
 ecow               = { workspace = true, public = true }
 foldhash           = { workspace = true, public = true }
-hashbrown          = { workspace = true, features = ["nightly"], public = true }
```

Comment:
Why the removal of `nightly` here?

Reply:
I moved the flag to the global `Cargo.toml`. If we would use `hashql` with another crate using `allocator-api2` indirectly we would get a compile error if the other crate does not take `allocator_api` into account. This globally enforces this.